### PR TITLE
Update __manifest__.py

### PR DESCRIPTION
**Pull request by jeremylwf**
removed price and currency, updated  website

### CODE REVIEW

File: omna/__manifest__.py [hunk 1]
```diff
@@ -1,37 +1,14 @@
...
-
-    # Categories can be used to filter modules in modules listing
-    # Check https://github.com/odoo/odoo/blob/13.0/odoo/addons/base/data/ir_module_category_data.xml
-    # for the full list
-    'category': 'Extra Tools',
-    'version': '0.1',
+    'name': 'OMNA Marketplace Connector',
+    'version': '13.0.0.1.0',
+    'category': 'Sales',
+    'summary': 'Shopify, Shopee, Lazada, Qoo10, MercadoLibre, Backmarket, Shipstation',
```
Comment (sanchojaf) [inline omna/__manifest__.py]: it is possible say " 'summary': 'Integrate: Shopify, Shopee, Lazada, Qoo10, MercadoLibre, Backmarket, Shipstation',"?

File: omna/__manifest__.py [hunk 2]
```diff
@@ -1,37 +1,14 @@
...
+    'description': 'Integrate global online marketplaces & web-stores with Odoo. Sync products, inventory and orders from multiple channels'"',
+    'author': 'OMNA Pte Ltd',
```
Comment (sanchojaf) [inline omna/__manifest__.py]: I prefer here "Cenit IO"

File: omna/__manifest__.py (hunk 1 above)
Comment (jeremylwf) [reply omna/__manifest__.py]: Yes, I propose. 'Integrations: Shopify, Shopee, Lazada, Qoo10, MercadoLibre, Backmarket, Shipstation'

File: omna/__manifest__.py (hunk 2 above)
Comment (jeremylwf) [reply omna/__manifest__.py]: for author?

File: omna/__manifest__.py (hunk 1 above)
Comment (sanchojaf) [reply omna/__manifest__.py]: > Yes, I propose. 'Integrations: Shopify, Shopee, Lazada, Qoo10, MercadoLibre, Backmarket, Shipstation'

could you make the change here with your wording?

**Review by sanchojaf — APPROVED** [review]
Looks good to me!

**Pull request merged**